stdlib/3/_ast.pyi

```diff
@@ -253,6 +270,14 @@ class Bytes(expr):
 class NameConstant(expr):
     value = ...  # type: Any
 
+if sys.version_info >= (3, 8):
+    class Constant(expr):
+        value = ...  # type: expr
```

Comment:
I guess this a an `expr`, because constants can be assigned an arbitrary expression? I mainly ask, because the old constant classes all had more concrete attribute types like 'str', 'float' (or `Any` in the case of `NameConstant`).